client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -210,74 +210,79 @@ export const RequestLineEditForm = ({
         </>
       }
       Middle={
-        <>
-          {isPacksEnabled && (
-            <Box display="flex" justifyContent="flex-end" alignItems="center">
-              <Switch
-                label={t('label.units')}
-                checked={isPacks}
-                onChange={(_event, checked) => setIsPacks(checked)}
-                size="small"
-              />
-              <Box paddingLeft={2} paddingRight={2}>
-                {t('label.packs')}
+        currentItem ? (
```

Comment:
oh woah big diff... actually just adds this conditional 🙃

Reply:
![Screenshot 2024-05-06 at 3 07 08 PM](https://github.com/msupply-foundation/open-msupply/assets/5456533/4c095beb-199e-496c-ab67-bbb7f8c91fff)

Not so bad with "Hide whitespace" selected 😉